spec/suites/layer/tile/TileLayerSpec.js

```diff
@@ -431,6 +431,33 @@ describe('TileLayer', function () {
 				});
 			});
 		}
+
+		it('sets min/maxZoom appropriately with detectRetina', function (done) {
+			const maxZoom = 1;
```

Comment:
```suggestion
			var maxZoom = 1;
```

if you replace `const` with `var` the tests should be running fine in IE10

Reply:
Right, simple mistake.